python_frank_energie/models.py

```diff
@@ -419,3 +419,59 @@ def from_userprices_dict(data: dict[str, str]) -> MarketPrices:
             electricity=PriceData(customerMarketPrices.get("electricityPrices")),
             gas=PriceData(customerMarketPrices.get("gasPrices")),
         )
+
+
+@dataclass
+class SmartBatteries:
+    """Collection of the users SmartBatteries"""
+
+    smart_batteries: list[SmartBattery]
+
+    @staticmethod
+    def from_dict(data: dict[str, str]) -> SmartBatteries:
+        """Parse the response from the smartBatteries query."""
+        _LOGGER.debug("SmartBatteries %s", data)
+
+        if errors := data.get("errors"):
+            raise RequestException(errors[0]["message"])
+
+        payload = data.get("data")
+        if not payload:
+            raise RequestException("Unexpected response")
+
+        return SmartBatteries(
+            smart_batteries=[
+                SmartBatteries.SmartBattery.from_dict(smart_battery)
+                for smart_battery in payload.get("smartBatteries")
+            ],
+        )
+
+    @dataclass
+    class SmartBattery:
```

Comment:
"""Collection of the users SmartBatteries."""
```suggestion
    class SmartBattery:
        """SmartBattery model."""
```